package.json

```diff
@@ -25,7 +29,8 @@
   },
   "homepage": "https://github.com/fraserxu/electron-pdf",
   "devDependencies": {
-    "standard": "^6.0.6"
+    "jasmine": "^2.5.2",
```

Comment:
Hmm, why `jasmine`? Haven't seen it being used for a while, haha. 

Have you ever looked into [ava.js](https://github.com/avajs/ava) or [tape](https://github.com/substack/tape) before, me personally more keen to use one of them over [mochajs](https://mochajs.org/) or jasmine. What do you think?